tsl/test/expected/dist_hypertable-11.out

```diff
@@ -192,12 +192,22 @@ INSERT INTO disttable VALUES
 INSERT INTO disttable VALUES
        ('2017-01-01 06:01', 1, 1.1),
        ('2017-01-01 09:11', 3, 2.1),
+       ('2017-01-01 09:21', 3, 2.2),
```

Comment:
How do we know this is enough? Is it purely incidental?

Reply:
That's why this is a draft. Will try with a few variations if this doesn't work.

Reply:
@erimatnor I re-ran the PG 11/12/13 regression tests here multiple times and `dist_hypertable` test is holding up. Looks like this is a good start.

Reply:
i'm having doubts this will fix it, have a look at https://github.com/timescale/timescaledb/issues/3139
you can reproduce the error locally by reverting PARALLEL_GROUP_SIZE and max_connection change in https://github.com/timescale/timescaledb/commit/202cceee56d1f8fb5844b0d7be6da2f9d65120f8 
and lowering max_connections to 50

Reply:
@svenklemm as you can see, this specific failure is a PLAN failure and doesn't seem to have anything to do with max connections.

Reply:
i could reproduce it doing the wrong plan with those changed settings

Reply:
Thanks for confirming. As I said, we can always iterate this further if issues persist on CI.